src/responder/common/cache_req/cache_req.c

```diff
@@ -974,6 +974,13 @@ static void cache_req_search_domains_done(struct tevent_req *subreq)
     case ERR_ID_OUTSIDE_RANGE:
```

Comment:
Btw, is it ok this way we can translate `ERR_ID_OUTSIDE_RANGE` to `ERR_OFFLINE`?

Reply:
What will be value of `dp_success` in case of single domain and ERR_ID_OUTSIDE_RANGE?

Reply:
dp_success will be false only if any of the dp requests fails (meaning we were unable to get requested data). If we don't have anything to return and there is nothing left to check (ENOENT && check_next == false) we return ERR_OFFLINE meaning there is nothing in the cache and dp failed. ERR_ID_OUTSIDE_RANGE is pretty much the same as ENOENT, it is used inside cache_req to shortcut if possible but it means ENOENT in this case. So I believe the code is correct.